doc/src/modules/physics/mechanics/masses.rst

```diff
@@ -139,6 +139,38 @@ The mass is specified exactly as is in a particle. Similar to the
 ``.masscenter`` must be specified. The reference frame is stored in an analogous
 fashion and holds information about the body's orientation and angular velocity.
 
+Loads
+=====
+
+In :mod:`sympy.physics.mechanics` loads can either be represented with tuples or
+with the dedicated classes :class:`~.Force` and :class:`~.Torque`. Generally the
+first argument (or item in the case of a tuple) is the location of the load. The
+second argument is the vector. In the case of a force the first argument is a
+point and the second a vector.
```

Comment:
It's perhaps way too late, but I wonder why mechanics didn't use geometric objects to do some of this, like using `Ray` for a force whose p1 is the point of application and the vector from p1 to p2 giving the direction and the magnitude, e.g. `force = lambda pt, mag, ang: Ray((0,0), (mag,0)).rotate(ang).translate(*pt)` for a force of magnitude 2 in the 45 deg direction from (1,2): `force((1,2),2,pi/4)`.

Reply:
Did that model exist back in 2010 or so? Also, everything in mechanics is based off the physics Vector which is how we manage these things.

Reply:
Looks like point.py might be as old as 2007. The answer is that we decided to implement a Vector object and we wrote the necessary methods for that object. I supposed we could have built Vector on top of some existing geometry tools back then, but it didn't happen.